salt/metalk8s/salt/minion/files/minion-99-metalk8s.conf.j2

```diff
@@ -8,3 +8,10 @@ use_superseded:
   - module.run
 
 log_level_logfile: info
+
+saltenv: {{ saltenv }}
+
+# Prepare volume at startup (required for loop devices persistency).
+startup_states: sls
+sls_list:
+  - /metalk8s/volumes/prepared
```

Comment:
Is there a way to specify the `saltenv` here instead of setting it globally for the minion?

Reply:
@TeddyAndrieux any idea about that?

Reply:
From my last search I not found any way to do this. 

I would prefer also to not define it in the minion config, but after some reflection as it's the version of the minion and when we will upgrade/downgrade this node we will also update the minion configuration.... for me it's ok to have it here. (And btw it's maybe also mandatory for startup_states :stuck_out_tongue: )

Reply:
Also with this change if salt handle the mount of the volumes that mean if we do a downgrade to something before 2.4 we will no longer mount our volumes (maybe we don't care, maybe not :nerd_face:)

Reply:
`kubelet` manages mounts, not Salt.

Reply:
Meh, it's fine I guess... in the end, the minion should be in that saltenv in the first place anyway.

Reply:
Ok, then let's keep it as is (thanks for the feedback @TeddyAndrieux)